assets/xml/objects/object_dekunuts.xml

```diff
@@ -1,42 +1,55 @@
-        <DList Name="object_dekunuts_DL_001E50" Offset="0x1E50" />
-        <DList Name="object_dekunuts_DL_001F50" Offset="0x1F50" />
-        <Texture Name="object_dekunuts_Tex_001FD8" OutName="tex_001FD8" Format="rgba16" Width="16" Height="32" Offset="0x1FD8" />
-        <Limb Name="object_dekunuts_Standardlimb_0023D8" Type="Standard" EnumName="OBJECT_DEKUNUTS_LIMB_01" Offset="0x23D8" />
-        <Limb Name="object_dekunuts_Standardlimb_0023E4" Type="Standard" EnumName="OBJECT_DEKUNUTS_LIMB_02" Offset="0x23E4" />
-        <Limb Name="object_dekunuts_Standardlimb_0023F0" Type="Standard" EnumName="OBJECT_DEKUNUTS_LIMB_03" Offset="0x23F0" />
-        <Limb Name="object_dekunuts_Standardlimb_0023FC" Type="Standard" EnumName="OBJECT_DEKUNUTS_LIMB_04" Offset="0x23FC" />
-        <Limb Name="object_dekunuts_Standardlimb_002408" Type="Standard" EnumName="OBJECT_DEKUNUTS_LIMB_05" Offset="0x2408" />
-        <Limb Name="object_dekunuts_Standardlimb_002414" Type="Standard" EnumName="OBJECT_DEKUNUTS_LIMB_06" Offset="0x2414" />
-        <Limb Name="object_dekunuts_Standardlimb_002420" Type="Standard" EnumName="OBJECT_DEKUNUTS_LIMB_07" Offset="0x2420" />
-        <Limb Name="object_dekunuts_Standardlimb_00242C" Type="Standard" EnumName="OBJECT_DEKUNUTS_LIMB_08" Offset="0x242C" />
-        <Limb Name="object_dekunuts_Standardlimb_002438" Type="Standard" EnumName="OBJECT_DEKUNUTS_LIMB_09" Offset="0x2438" />
-        <Skeleton Name="object_dekunuts_Skel_002468" Type="Normal" LimbType="Standard" LimbNone="OBJECT_DEKUNUTS_LIMB_NONE" LimbMax="OBJECT_DEKUNUTS_LIMB_MAX" EnumName="object_dekunuts_Limbs" Offset="0x2468" />
-        <Animation Name="object_dekunuts_Anim_00259C" Offset="0x259C" />
-        <Animation Name="object_dekunuts_Anim_002950" Offset="0x2950" />
-        <Animation Name="object_dekunuts_Anim_002A5C" Offset="0x2A5C" />
-        <Animation Name="object_dekunuts_Anim_002BD4" Offset="0x2BD4" />
-        <Animation Name="object_dekunuts_Anim_002DD4" Offset="0x2DD4" />
-        <Animation Name="object_dekunuts_Anim_002FA4" Offset="0x2FA4" />
-        <Animation Name="object_dekunuts_Anim_003180" Offset="0x3180" />
-        <Animation Name="object_dekunuts_Anim_00326C" Offset="0x326C" />
-        <Animation Name="object_dekunuts_Anim_0033E4" Offset="0x33E4" />
-        <Animation Name="object_dekunuts_Anim_003780" Offset="0x3780" />
+        <Animation Name="gDekuScrubSpitAnim" Offset="0x168" /> <!-- Original name is "dn_atack" -->
+
+        <!-- Deku Scrub Limb DisplayLists -->
+        <DList Name="gDekuScrubRightFootDL" Offset="0xA00" />
+        <DList Name="gDekuScrubRightLegDL" Offset="0xAA8" />
+        <DList Name="gDekuScrubLeftFootDL" Offset="0xB40" />
+        <DList Name="gDekuScrubLeftLegDL" Offset="0xBE8" />
+        <DList Name="gDekuScrubSnoutDL" Offset="0xC80" />
+        <DList Name="gDekuScrubCrestDL" Offset="0xD90" />
+        <DList Name="gDekuScrubHairDL" Offset="0xE30" />
```

Comment:
I don't think "hair" is a very good description for this, even if it does seem to be coming out of its head. Deku Link seems to have actual hair, which is pretty weird, but every other Deku it's more a sort of leafy shawl or cloak or hood?

Reply:
One of the reasons I named it "hair" is because of this:
![NKZE01_2022-04-21_15-37-59](https://user-images.githubusercontent.com/12245827/164562852-8aae9ac0-89a6-490e-a87f-b40579c29bba.png)
To make the Deku Scrubs look like this, they just rotate this limb upwards. Would it make sense for a "cloak" or "shawl" or "hood" to stick straight up like this? An argument *against* hair is that they just wanted some visual variety, so they did the rotation without caring if it made any logical sense. I'm totally sympathetic to this line of thinking, but I'm curious if seeing this screenshot changes your opinion on my naming. If it doesn't, that's totally understandable, and I'll change it.

Reply:
Ah, I hadn't realised it was the same thing. I think basically I don't want to call it that when it looks like leaves.... "Headdress"?

Reply:
Sure, that works too. Let's go with that for now.